### PR TITLE
feat(DSRE-1774) Grant Google Ads access to desktop conv events view

### PR DESCRIPTION
**Pull request by kwindau**
## Description

This PR is intended to enable the Google ads account to have read access to the view
`mozdata.mozilla_org.desktop_conversion_events`

## Related Tickets & Documents
* [DSRE-1774](https://mozilla-hub.atlassian.net/browse/DSRE-1774)

**Reviewer, please follow [this checklist](https://github.com/mozilla/bigquery-etl/blob/main/.github/reviewer_checklist.md)**

[DSRE-1774]: https://mozilla-hub.atlassian.net/browse/DSRE-1774?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-5547)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/mozilla_org/desktop_conversion_events/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+friendly_name: Desktop Conversion Events
```
Comment (whd) [inline sql/moz-fx-data-shared-prod/mozilla_org/desktop_conversion_events/metadata.yaml:1]: I'd recommend copying the minimum set of metadata here, which for the matter at hand is just `workgroup_access`. In particular I don't think you should copy `references` as that's typically auto-generated metadata.

File: sql/moz-fx-data-shared-prod/mozilla_org/desktop_conversion_events/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+description: |-
+  Desktop conversion events sent to Google Ads for optimizing marketing campaigns
+owners: []
```
Comment (whd) [inline sql/moz-fx-data-shared-prod/mozilla_org/desktop_conversion_events/metadata.yaml:4]: Should this be empty?

**Review by whd — APPROVED** [review]
I don't have any specific knowledge of this system other than what I read from https://support.google.com/google-ads/answer/13763054?hl=en, which I translated into the Jira ticket and which seems to match this PR.

File: sql/moz-fx-data-shared-prod/mozilla_org/desktop_conversion_events/metadata.yaml (hunk 1 above)
Comment (kwindau) [reply sql/moz-fx-data-shared-prod/mozilla_org/desktop_conversion_events/metadata.yaml:1]: got it, will fix that, thanks!

**Pull request merged**